src/tools/inner_node.rs

```diff
@@ -100,3 +108,22 @@ impl InnerNode {
         self.node.shut_down().await
     }
 }
+
+fn decode_to_vec(base58str: &str, size: usize) -> bs58::decode::Result<Vec<u8>> {
+    let mut bytes = bs58::decode(base58str)
+        .with_alphabet(bs58::Alphabet::RIPPLE)
+        .into_vec()?;
+    bytes.remove(0);
```

Comment:
Just wondering, why are we removing the first byte here?

Reply:
The first byte is an extra information to differentiate between different things in rippled (so that all private keys start with same letter after serialization, same applies to public keys, seeds, I think accounts and transactions as well).

Reply:
Maybe we could leave a small comment for that info 🙂

Reply:
Done.